pkgs/top-level/aliases.nix

```diff
@@ -346,6 +346,7 @@ mapAliases ({
   gnuradio-limesdr = gnuradio3_7.pkgs.limesdr; # added 2019-05-27, changed 2020-10-16
   gnuradio-rds = gnuradio3_7.pkgs.rds; # added 2019-05-27, changed 2020-10-16
   gnuradio-osmosdr = gnuradio3_7.pkgs.osmosdr; # added 2019-05-27, changed 2020-10-16
+  gnuradio3_7 = throw "gnuradio3_7 has been removed because it required Python 2"; # added 2022-01-16
```

Comment:
```suggestion
  gnuradio3_7 = throw "gnuradio3_7 has been removed because it requires Python 2"; # added 2022-01-16
```

Reply:
Actually I disagree. It could be a patch release of 3.7 is released that works with Python 2. In that case, what I wrote still applies and what you wrote no longer does ;-)
Though I should have written "was removed".